### PR TITLE
Nz/3592 subform view

### LINKED ISSUE #3592: [Feature] Update sub form views
**Description and related issues -**
Update sub form views to now be indented with a blue line, like Assurances and Compliance. Create a reusable component that can be used in all of the following locations:

Standards and Conditions

Outcomes and Metrics

Private Contractor Costs

Other Funding

**Mockups or link to Figma**
https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2021?node-id=0%3A16744

### This task is done when…

- [x] the styling looks like the figma mock up
- [x] styling is used in Standards and Conditions
- [x] styling is used in Outcomes and Metrics
- [x] styling is used in Private Contractor Costs
- [x] styling is used in Other Funding
- [x] tests have been updated for styling change

### PR DESCRIPTION
**Pull request by Sun-Mountain**
Resolves #3592 

**Description-**
Update sub form views to now be indented with a blue line, like Assurances and Compliance.

**This pull request changes...**

- Subform views have been updated thru out the application and export view.

**This pull request was tested in the follow ways…**

- Changed subform views when applicable in the application.
- Changed subform views in export view.

### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [x] Code has been reviewed by someone other than the original author
- [x] QA has verified the accessibility and functionality related to the change
- [x] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

File: web/src/containers/activity/ContractorResource/ContractorResourceReview.js [hunk 1]
```diff
@@ -46,11 +46,27 @@ const ContractorResourceReview = ({
         <li>
           <strong>Total Contract Cost:</strong> <Dollars>{totalCost}</Dollars>
         </li>
-        {apdFFYs.map(ffy => (
-          <li key={ffy}>
-            <strong>FFY {ffy} Cost:</strong> <Dollars>{years[ffy]}</Dollars>
-          </li>
-        ))}
+        {apdFFYs.map(ffy => {
```
Comment (thetif) [inline web/src/containers/activity/ContractorResource/ContractorResourceReview.js:49]: a shortcut you could do here is to do 
```
{apdFFYs.map(ffy => hourly.useHourly === false ? 
```
when you create an unnamed function it automatically returns

File: integrationTests/cypress/integration/02-apd/01-apd-basics.spec.js [hunk 2]
```diff
@@ -347,6 +348,7 @@ describe('APD Basics', { tags: ['@apd', '@default'] }, () => {
         .check({ force: true });
       cy.findByRole('button', { name: /Save/i }).click();
 
+      cy.log("LOL")
```
Comment (thetif) [inline integrationTests/cypress/integration/02-apd/01-apd-basics.spec.js:351]: do you still need all of these log statements

File: integrationTests/cypress/page-objects/activity-page.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ class ActivityPage {
     cy.contains(`Total Contract Cost: $${addCommas(totalCosts)}`).should(
       'exist'
     );
-    this.checkFFYcosts({ years, FFYcosts });
+    // this.checkFFYcosts({ years, FFYcosts });
```
Comment (thetif) [inline integrationTests/cypress/page-objects/activity-page.js:158]: remove commented out code if it's not needed

**Review by thetif — APPROVED** [review]
looks good, just a couple of minor suggestions

File: integrationTests/cypress/integration/02-apd/01-apd-basics.spec.js (hunk 2 above)
Comment (Sun-Mountain) [reply integrationTests/cypress/integration/02-apd/01-apd-basics.spec.js:351]: Haha, nope. Removing.

**Review by thetif — APPROVED** [review]

**Review by aricka-lewis — APPROVED** [review]
Looks good to me

**Review by jeromeleecms — COMMENTED** [review]
@Sun-Mountain Having a data entry issue: is this related to this PR? 

image

When I put in an invalid date, there is no indicator for what is a valid data (or even the inline error where it says to type in an valid date).

For the export view with the contractor view, is it supposed to look like like the indented version? Currently it looks like this: 

image

The hourly version is consistent with the builder and the export view.

Checking with @akuas and @aricka-lewis for the design question.

**Comment by jeromeleecms** [comment]
Per Stand-up Today (@Sun-Mountain @akuas let me know if this is not what you understood to be the changes):
1) Form validation is not tied to this PR. Changes will happen in another PR. ✔️ 
2) For private contractor costs, the indentation should only happen when the hourly rate is checked "yes." When it is not, there should be no indents and should appear like the screen shot (in both the export and the builder views):

image

3) Cost allocation indentation should show in the export view.

Follow-up items:
1) Branching logic should have designs that represent both states (when Yes / No is selected).
2) Product to review validation Mural to ensure we cover the validation cases
3) Design to review consistency of when we use "FFY 2021 Costs" vs just "FFY 2021"

**Comment by Sun-Mountain** [comment]
> Per Stand-up Today (@Sun-Mountain @akuas let me know if this is not what you understood to be the changes):
> 
> 1. Form validation is not tied to this PR. Changes will happen in another PR. ✔️
> 2. For private contractor costs, the indentation should only happen when the hourly rate is checked "yes." When it is not, there should be no indents and should appear like the screen shot (in both the export and the builder views):
> 
> image
> 
> 3. Cost allocation indentation should show in the export view.
> 
> Follow-up items:
> 
> 1. Branching logic should have designs that represent both states (when Yes / No is selected).
> 2. Product to review validation Mural to ensure we cover the validation cases
> 3. Design to review consistency of when we use "FFY 2021 Costs" vs just "FFY 2021"

@jeromeleecms I have applied the changes you requested!

**Review by jeromeleecms — APPROVED** [review]
Changes look good! :tada:

**Pull request merged**